hardening/host-os/oscap/test.py

```diff
@@ -15,15 +16,23 @@
 ds = util.get_datastream()
 
 unique_name = util.get_test_name().lstrip('/').replace('/', '-')
-new_ds = Path(f'/var/tmp/contest-{unique_name}')
+
```

Comment:
The empty line is not needed there

Reply:
It's not very visible from the quoted code, but the context is:
```python
unique_name = util.get_test_name().lstrip('/').replace('/', '-')

# persistent across reboots
tmpdir = Path(f'/var/tmp/contest-{unique_name}')

new_ds = tmpdir / 'modified_datastream.xml'
```
I added the newline to prevent somebody associating the "persistent across reboots" comment with the `new_ds` line. But I guess I can move it closer, the comment wouldn't be wrong per se.